test/lib/ufe/testDeleteCmd.py

```diff
@@ -259,11 +369,12 @@ def testDeleteArgs(self):
         ball35PathString = "|transform1|proxyShape1,/Room_set/Props/Ball_35"
 
         ufeObs.reset()
+        
         cmds.delete(
             ball35PathString, ball34PathString, "|pSphere1|pSphereShape1")
-
-        # We deleted 3 items.
-        self.assertEqual(ufeObs.nbDeleteNotif(), 3)
+        
+        # We deleted 1 items.
```

Comment:
Again, here I believe that we're only testing the delete of a Maya object, which makes me uncomfortable.  We really have to test that USD objects are deleted.